conf/ocsci/dr_workload.yaml

```diff
@@ -0,0 +1,2 @@
+ENV_DATA:
+  dr_workload_repo_url: "https://github.com/shylesh/ocm-ramen-samples.git"
```

Comment:
this repo will only work on the given  cluster name if diff app are not deployed https://github.com/shylesh/ocm-ramen-samples/blob/main/subscriptions/busybox/drpc.yaml#L9

Reply:
@prsurve Waiting for clarity on this from Shyam.

Reply:
We will handle this as and when required.